commit 4cbe00ff5ef69e7d83af5b7f0762a765ae3ceefd (refs/changes/40/5140/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-11-26T14:15:37-08:00 (11 months ago)
    
    TRITON-981 get lagging Triton components on the latest v0.10.x
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/Makefile b/Makefile
index e0ea01e..6d203a4 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2017, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -28,7 +28,7 @@ SMF_MANIFESTS_IN = smf/manifests/backupserver.xml.in \
 # Variables
 #
 
-NODE_PREBUILT_VERSION   := v0.10.26
+NODE_PREBUILT_VERSION   := v0.10.48
 NODE_PREBUILT_TAG       := zone
 # Allow building on a SmartOS image other than sdc-multiarch/13.3.1.
 NODE_PREBUILT_IMAGE=b4bdc598-8939-11e3-bea4-8341f6861379
diff --git a/package.json b/package.json
index e11973f..4db2cea 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "sdc-manatee",
-    "version": "1.0.1",
+    "version": "1.0.2",
     "private": true,
     "description": "H/A Postgres Minder",
     "author": "Joyent (joyent.com)",
